### DISCUSSION TITLE
Package Build Always Fails at Login with GitHub

### DISCUSSION THREAD

**Discussion by JamesLloydMesser** [community/community#139211] [Programming Help]
### Select Topic Area

Question

### Body

I'm new at this.  I have a simple package that I am building through Anaconda Learning.  It keeps failing with setting up the interface with GitHub. I get the CredentialHelperSelector (Dialog Box).  I select the Manager (first of the two).  I select browser login.  The browser opens with a URL of http://[127.0.0.1:54852/?code=040c6705ddb7902039d7&state=7262a5f53ee5436a8c990f643903b7b0](http://127.0.0.1:54852/?code=040c6705ddb7902039d7&state=7262a5f53ee5436a8c990f643903b7b0), but I do not get a dialog box to enter my password. I done it a several times now.

I also have tried the token at least twice.  I get Congratulations, you're all set! Your device is now connected.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @JamesLloydMesser ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
